[2025-07-21T17:27:04.330+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: complex_failure_dag.start_task manual__2025-07-21T17:27:04.079061+00:00 [queued]>
[2025-07-21T17:27:04.341+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: complex_failure_dag.start_task manual__2025-07-21T17:27:04.079061+00:00 [queued]>
[2025-07-21T17:27:04.341+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2025-07-21T17:27:04.358+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): start_task> on 2025-07-21 17:27:04.079061+00:00
[2025-07-21T17:27:04.367+0000] {standard_task_runner.py:57} INFO - Started process 192 to run task
[2025-07-21T17:27:04.371+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'complex_failure_dag', 'start_task', 'manual__2025-07-21T17:27:04.079061+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/complex_failure_dag.py', '--cfg-path', '/tmp/tmpxvo3afkv']
[2025-07-21T17:27:04.373+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask start_task
[2025-07-21T17:27:04.430+0000] {task_command.py:416} INFO - Running <TaskInstance: complex_failure_dag.start_task manual__2025-07-21T17:27:04.079061+00:00 [running]> on host 2cf9a9b2cf2c
[2025-07-21T17:27:04.500+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='metadata_test' AIRFLOW_CTX_DAG_ID='complex_failure_dag' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-07-21T17:27:04.079061+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-21T17:27:04.079061+00:00'
[2025-07-21T17:27:04.501+0000] {task_logging_plugin.py:37} INFO - [Task Metadata] (running): 
{'event': 'TASK_RUNNING', 'timestamp': '2025-07-21T17:27:04.501552', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T17:27:04.079061+00:00', 'task_id': 'start_task', 'execution_date': '2025-07-21 17:27:04.079061+00:00', 'try_number': 1, 'start_date': '2025-07-21 17:27:04.330608+00:00', 'end_date': 'None', 'state': 'running', 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c', 'log_url': 'http://localhost:8080/dags/complex_failure_dag/grid?dag_run_id=manual__2025-07-21T17%3A27%3A04.079061%2B00%3A00&task_id=start_task&tab=logs'}
[2025-07-21T17:27:04.501+0000] {task_logging_plugin.py:41} INFO - [Task Metadata]-logs**** (running): 
{'event': 'TASK_RUNNING', 'timestamp': '2025-07-21T17:27:04.501552', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T17:27:04.079061+00:00', 'task_id': 'start_task', 'execution_date': '2025-07-21 17:27:04.079061+00:00', 'try_number': 1, 'start_date': '2025-07-21 17:27:04.330608+00:00', 'end_date': 'None', 'state': 'running', 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c', 'log_url': 'http://localhost:8080/dags/complex_failure_dag/grid?dag_run_id=manual__2025-07-21T17%3A27%3A04.079061%2B00%3A00&task_id=start_task&tab=logs'}
[2025-07-21T17:27:04.512+0000] {task_graph_plugin.py:38} INFO - 
[task_graph_logs] Task Graph for DAG 'complex_failure_dag':
  Task: start_task, Upstream: [], Downstream: ['sometimes_fail_task', 'long_running_task'], Operator: SerializedBaseOperator
  Task: sometimes_fail_task, Upstream: ['start_task'], Downstream: ['final_cleanup'], Operator: SerializedBaseOperator
  Task: long_running_task, Upstream: ['start_task'], Downstream: ['final_cleanup'], Operator: SerializedBaseOperator
  Task: final_cleanup, Upstream: ['sometimes_fail_task', 'long_running_task'], Downstream: ['guaranteed_failure'], Operator: SerializedBaseOperator
  Task: guaranteed_failure, Upstream: ['final_cleanup'], Downstream: [], Operator: SerializedBaseOperator

[2025-07-21T17:27:04.513+0000] {task_graph_plugin.py:42} INFO - 
[task_graph_logs] Task Graph for DAG 'complex_failure_dag':
  Task: start_task, Upstream: [], Downstream: ['sometimes_fail_task', 'long_running_task'], Operator: SerializedBaseOperator
  Task: sometimes_fail_task, Upstream: ['start_task'], Downstream: ['final_cleanup'], Operator: SerializedBaseOperator
  Task: long_running_task, Upstream: ['start_task'], Downstream: ['final_cleanup'], Operator: SerializedBaseOperator
  Task: final_cleanup, Upstream: ['sometimes_fail_task', 'long_running_task'], Downstream: ['guaranteed_failure'], Operator: SerializedBaseOperator
  Task: guaranteed_failure, Upstream: ['final_cleanup'], Downstream: [], Operator: SerializedBaseOperator

[2025-07-21T17:27:04.513+0000] {logging_mixin.py:154} INFO - Starting complex failure DAG
[2025-07-21T17:27:04.514+0000] {logging_mixin.py:154} INFO - Current time: 2025-07-21 17:27:04.514050
[2025-07-21T17:27:24.533+0000] {logging_mixin.py:154} INFO - Start task completed successfully
[2025-07-21T17:27:24.534+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-07-21T17:27:24.551+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=complex_failure_dag, task_id=start_task, execution_date=20250721T172704, start_date=20250721T172704, end_date=20250721T172724
[2025-07-21T17:27:24.563+0000] {task_logging_plugin.py:37} INFO - [Task Metadata] (success): 
{'event': 'TASK_SUCCESS', 'timestamp': '2025-07-21T17:27:24.563592', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T17:27:04.079061+00:00', 'task_id': 'start_task', 'execution_date': '2025-07-21 17:27:04.079061+00:00', 'try_number': 2, 'start_date': '2025-07-21 17:27:04.330608+00:00', 'end_date': '2025-07-21 17:27:24.551151+00:00', 'state': <TaskInstanceState.SUCCESS: 'success'>, 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c', 'log_url': 'http://localhost:8080/dags/complex_failure_dag/grid?dag_run_id=manual__2025-07-21T17%3A27%3A04.079061%2B00%3A00&task_id=start_task&tab=logs'}
[2025-07-21T17:27:24.564+0000] {task_logging_plugin.py:41} INFO - [Task Metadata]-logs**** (success): 
{'event': 'TASK_SUCCESS', 'timestamp': '2025-07-21T17:27:24.563592', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T17:27:04.079061+00:00', 'task_id': 'start_task', 'execution_date': '2025-07-21 17:27:04.079061+00:00', 'try_number': 2, 'start_date': '2025-07-21 17:27:04.330608+00:00', 'end_date': '2025-07-21 17:27:24.551151+00:00', 'state': <TaskInstanceState.SUCCESS: 'success'>, 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c', 'log_url': 'http://localhost:8080/dags/complex_failure_dag/grid?dag_run_id=manual__2025-07-21T17%3A27%3A04.079061%2B00%3A00&task_id=start_task&tab=logs'}
[2025-07-21T17:27:24.583+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-07-21T17:27:24.605+0000] {taskinstance.py:2778} INFO - 2 downstream tasks scheduled from follow-on schedule check
